### DISCUSSION TITLE
Batch erase with a selected region of influence.

### DISCUSSION THREAD

**Discussion by hornedpariah** [Sanster/IOPaint#182] [Ideas]
I have training data from a videogame and the subtitles are baked in, all my images are the same aspect ratio, subtitles are within a certain region. it would be nice to have a batch folder option with a select to to define an area of influence for removal.

**Comment by SirTrippsalot** [comment]
Gui is another sticky issue that would be nice to clean out

**Comment by transitgrave** [comment]
I too would love to see this functionality. I have been looking for a watermark solution to clean up ModelScope text2video. Thanks :)

**Reply by navacademia** [reply]
This is exactly what I came searching for. 
Especially considering the project allows you to save and store the mask. 
Static watermarks from videos could be removed if the file was split into individual frames.

**Comment by werran2** [comment]
great idea
